package.json

```diff
@@ -33,7 +33,8 @@
     "vendor": "webpack --config ./config/webpack.vendor.config.js",
     "publish-demo": "webpack --config ./config/webpack.sandbox.config.js",
     "test": "jest --coverage",
-    "test-ci": "jest && codecov"
+    "test-ci": "jest && codecov",
+    "eslint-fix": "find $PWD/src -name '*.js' -exec eslint --fix {} \\;"
```

Comment:
This seems complicated.  I would prefer something like this:
`"eslint-fix": eslint -c .eslintrc ./src/ --fix`